packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -985,7 +985,7 @@ void main() {
     Material material = getMaterial(tester);
 
     // Test initial ExpansionTile properties.
-    expect(material.shape, const RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(4))));
+    expect(material.shape, collapsedShape);
```

Comment:
We might as well revert this one to reduce the diffs:

```suggestion
    expect(material.shape, const RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(4))));
```